data/json/items/tool/workshop.json

```diff
@@ -87,6 +87,36 @@
       }
     ]
   },
+  {
+    "type": "GENERIC",
+    "id": "grinder_cutoff_disc",
```

Comment:
Both of discs seems to be unused too?

Reply:
Yeah, I didn't add a spawn location form them, since there is no way to combine them with the angle grinder tool yet.